get_homepage_data.py

```diff
@@ -75,3 +91,17 @@ def get_homepage_standings():
         }
         standings_docs.append(doc)
     return standings_docs, standings_columns
+
+
+def get_current_scores(start_date, final_date):
+    score_dict = {}
+    espn_score_data = requests.get(
+        f"http://site.api.espn.com/apis/site/v2/sports/football/nfl/scoreboard?dates={start_date}-{final_date}").json()
```

Comment:
Not super important but it might be good to `try/except` this at some point so if there's an error retrieving the JSON, we don't break the whole page and instead just get the teams without scores.

Reply:
Also, I'm guessing you probably looked into this but is there an attribute in the JSON returned that tells whether a game is final or not? Would be interesting to maybe bold the text or something once a game goes final.